Python/tests/test_client.py

```diff
@@ -3,13 +3,15 @@
 Copyright 2020 Microsoft
 """
 from unittest.mock import Mock, patch
-from microsoft_bonsai_api.simulator.client import BonsaiClientConfig, BonsaiClient
-from microsoft_bonsai_api.simulator.generated.models import (
-    SimulatorInterface,
-    SimulatorState,
-)
+
+import pytest
```

Comment:
I don't think it's a good idea to take a dependency on pytest when we're already using unittest for this test case. Those are two different test frameworks. Plus, pytest is not built in to the stdlib, so this introduces another external dependency that previously didn't exist.

Reply:
Thanks for your comment! Actually I used `pytest` because in the `setup.py` it is said to be the test suite for the package (see [here](https://github.com/microsoft/microsoft-bonsai-api/blob/5f6e262944805a9180c54ed8c8f1a7ebd17d455d/Python/setup.py#L37)).
However, if you think it's better not to use it I can amend the PR and remove it so that only point 2. is addressed.

Reply:
Ah, I missed that. It looks like there are other modules that already import pytest. That's a little weird because it's also importing unittest. Normally those aren't both used in the same package.

In any case, it looks like you're not adding a new dependency here, so ignore my comment above.